string_slicing_operations.py

```diff
@@ -0,0 +1,33 @@
+# Enter string value
+a = input("Enter a string value:")
+
+# Confirm the entry
+print (f"You've entered: {a}")
+
+#Find length of string
+n = len(a)
+
+# Slice after removing first and last character
+b = a[1:n-1]
+print(f"Slice of '{a}' after removing first and last characters is '{b}'")
+
+#Slice after removing the first 2 characters
+c = a[2:n]
```

Comment:
This works because "n" here is not printed in slicing. But be careful accessing an index out of the boundary of a string.